pom.xml

```diff
@@ -64,7 +64,7 @@
         <gravitee-license-node.version>1.3.1</gravitee-license-node.version>
         <gravitee-node.version>4.0.0-alpha.2</gravitee-node.version>
         <gravitee-notifier-api.version>1.4.3</gravitee-notifier-api.version>
-        <gravitee-plugin.version>2.0.0-alpha.4</gravitee-plugin.version>
+        <gravitee-plugin.version>2.0.0-archi-253-plugin-override-SNAPSHOT</gravitee-plugin.version>
```

Comment:
To be changed, when plugin is out.

Reply:
you should first merge gravitee plugin, bump the version here then merge this PR.

Reply:
that's what I meant, it was a self note